storage/innobase/handler/ha_innodb.cc

```diff
@@ -12867,11 +12867,6 @@ bool create_table_info_t::create_option_encryption_is_valid() const {
   bool table_is_keyring =
       Encryption::is_keyring(m_create_info->encrypt_type.str);
 
-  if (table_is_keyring && !m_allow_file_per_table) {
-    my_printf_error(ER_KEYRING_ILLEGAL_ENCRYPTION_OPTION,
-                    "InnoDB: KEYRING requires innodb_file_per_table.", MYF(0));
-    return (false);
-  }
```

Comment:
After this PR, do we allow creation of KEYRING general tablespaces? ie CREATE TABLESPACE ts1 ADD DATAFILE ts1.ibd ENCRYPTION=KEYRING?

Also, does this change mean, we now allow keyring tables in general tablespaces? Previously it wasn't allowed? :thinking: